src/components/TradeZone/Feed.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <v-container fluid>
+
```

Comment:
You'll have to remove this extra space

Reply:
@GadoliniumPotassium Hope you don't mind I went ahead and did that for you. Re-requesting reviews now